### PR TITLE
Remove reference to fields becoming private in EAP.

### LINKED ISSUE #1941: EAP build failing

### PR DESCRIPTION
**Pull request by ivanporty**
Fixes #1941.

Repository fields are private in upcoming releases, so we can use shorter version of constructor that takes two public repositories by default. I quickly tested and it seems to work fine.

### CODE REVIEW

File: google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudApiMavenService.java [hunk 1]
```diff
@@ -45,13 +44,9 @@ static CloudApiMavenService getInstance() {
    * @return returns the {@link Version versions} of the BOMs
    */
   List<Version> getBomVersions() {
-    List<RemoteRepository> remoteRepositories =
-        ImmutableList.of(ArtifactRepositoryManager.MAVEN_CENTRAL_REPOSITORY);
```
Comment (etanshaul) [inline google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudApiMavenService.java]: is MAVEN_CENTRAL_REPOSITORY still used after this? wondering how it knows which remote repo to access with this update

Comment (ivanporty) [reply google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudApiMavenService.java]: They use two central repos - MAVEN_CENTRAL_REPOSITORY and JBOSS_COMMUNITY_REPOSITORY in this constructor. As an alternative, we may create Maven central repo directly from URL ourselves and use it instead so we can have more control, wdyt?

Comment (etanshaul) [reply google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudApiMavenService.java]: > we may create Maven central repo directly from URL ourselves and use it instead so we can have more control, wdyt?

Could you elaborate on that?

Comment (ivanporty) [reply google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudApiMavenService.java]: Sure, they create the repository like that in their code:
```java
private static final RemoteRepository MAVEN_CENTRAL_REPOSITORY = createRemoteRepository(
    "central", "http://repo1.maven.org/maven2/"
  );
```

we can take this approach and manually create the same repository in our code without relying on their selection of public repositories for `ArtifactRepositoryManager`

Comment (etanshaul) [reply google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudApiMavenService.java]: ah. I just looked at the code - so by default they use maven central and jboss..I think we want to exclude jboss.

so yeah - what you suggested sounds good to me

**Comment by ivanporty** [comment]
Their process in not so good it seems. Someone just changed that as part of using proxies - non-relevant change at all

**Pull request merged**